pkg/operator/ceph/csi/spec.go

```diff
@@ -726,7 +727,7 @@ func (r *ReconcileCSI) stopDrivers(ver *version.Info) error {
 	CephFSDriverName = fmt.Sprintf("%s.cephfs.csi.ceph.com", r.opConfig.OperatorNamespace)
 	NFSDriverName = fmt.Sprintf("%s.nfs.csi.ceph.com", r.opConfig.OperatorNamespace)
 
-	if !EnableRBD {
+	if !EnableRBD || EnableCSIOperator() {
 		logger.Info("CSI Ceph RBD driver disabled")
```

Comment:
to reduce confusion, might need a different message in case the csi operator is enabled

Reply:
added more logs